packages/react-dev-utils/getProcessForPort.js

```diff
@@ -0,0 +1,59 @@
+var chalk = require('chalk');
+var execSync = require('child_process').execSync;
+var path = require('path');
+
+var execOptions = { encoding: 'utf8' };
+
+function isProcessAReactApp(processCommand) {
+  return /^node .*react-scripts\/scripts\/start\.js\s?$/.test(processCommand);
+}
+
+function getProcessIdsOnPort(port) {
+  return execSync('lsof -i:' + port + ' -P -t', execOptions).match(/(\S+)/g);
```

Comment:
It's fine to not support Windows for "niceties" like this as long as there's a reasonable fallback.
Somebody can then add Windows-specific code in a future PR.